src/modules/deployment/utils.ts

```diff
@@ -42,14 +41,14 @@ export const getDefaultGroundAsset = (): Asset => ({
   actions: []
 })
 
-export async function getCID(files: ContentServiceFile[], shareRoot: boolean): Promise<string> {
+export async function getCID(files: ContentServiceFile[]): Promise<string> {
```

Comment:
Should we call this function getHash?

Reply:
This function didn't actually change! I just used `hasV1` when needed